src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.csproj

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Runtime.InteropServices.cs" />
+    <Compile Condition="'$(TargetsNetCoreApp)' == 'true'" Include="System.Runtime.InteropServices.netcoreapp.cs" />
```

Comment:
Originally, in #34322, we exposed the API universally and updated the baseline.
But @ericstj  recommended that we only expose the API for netcoreapp, because it is unlikely that the API will be implemented for UWP.

Reply:
I do not see a fundamental problem with implementing these APIs for UWP. They will be limited to what UWP allows you do, but that's ok. And we can always implement them by throwing PNSE if there is no better option.

Also, I think it is unlikely we would keep UWP as a separate targeting pack in the next major turn of the crank of that platform. I expect that we would just merge it with netcoreapp once that happens.

Reply:
> I do not see a fundamental problem with implementing these APIs for UWP. They will be limited to what UWP allows you do, but that's ok

It's my understanding that all of these are wrappers over LoadLibrary which is unavailable in UWP due to WACK. 
 Though I just had a look at https://docs.microsoft.com/en-us/uwp/win32-and-com/win32-apis and noticed LoadPackagedLibrary is available (as well as GetProcAddress and FreeLibrary).  I suppose I could imagine this being implemented on top of that for UWP.

>  it is unlikely we would keep UWP as a separate targeting pack in the next major turn of the crank

That's news to me.  Don't get me wrong I'd love to see it happen but I'm unaware of such POR.  I don't see it happening from a consumption side of things due to existing library ecosystem so as long as it remains a separate TFM we can provide a separate reference for it.

Reply:
There is no POR for next major turn of the crank for UWP today. I am just making a guess about what the POR is likely going to be once there is one.

Reply:
>  noticed LoadPackagedLibrary is available (as well as GetProcAddress and FreeLibrary

Right.

Reply:
Sure, sorry for the original poor advice.  That was my misunderstanding.